mediatimestamp/immutable.py

```diff
@@ -1267,20 +1258,15 @@ def normalise(self, rate_num, rate_den=1, rounding=ROUND_NEAREST, phase_offset=T
             else:
                 start = self.start.to_count(rate_num, rate_den, TimeRange.ROUND_DOWN)
 
-        if self.bounded_before() and rounding == TimeRange.PRESERVE_START:
-            phase_offset = self.start.to_phase_offset(rate_num, rate_den)
-        elif self.bounded_after() and rounding == TimeRange.PRESERVE_END:
-            phase_offset = self.end.to_phase_offset(rate_num, rate_den)
-
         if start is not None and not self.includes_start():
             start += 1
         if end is not None and self.includes_end():
             end += 1
 
         if start is not None:
-            start = Timestamp.from_count(start, rate_num, rate_den) + phase_offset
+            start = Timestamp.from_count(start, rate_num, rate_den)
```

Comment:
Does this mean the `phase_offset` kwarg needs to be removed? I don't think it does anything anymore.

Reply:
Thanks for spotting that. kwarg has been removed.